src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,413 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Primary UI component for user interaction
+ */
+const Pagination = ({ variant = 'primary',
+    borderVariant = 'black',
+    size = 'md',
+    rounded = 'md',
+    additionalClasses = [],
+    activeAdditionalClasses = [],
+    prevNextAdditionalClasses = [],
+    containerAdditionalClasses = [],
+    prev = 'Previous',
+    next = 'Next',
+    baseUrl = '/',
+    page = 1,
+    numOfPages = 1 }) => {
+    const sizeClass = {
+        sm: 'w-6 h-6 md:w-8 md:h-8 text-sm',
+        md: 'w-8 h-8 md:w-10 md:h-10 text-sm md:text-base',
+        lg: 'w-10 h-10 md:w-12 md:h-12 text-md md:text-lg',
+    };
+    const sizeClass2 = {
```

Comment:
Can we use another name than ...`2`?  I can't find out what does it mean `2` in this name

Reply:
we can use sizeClassPrevNext